internal/operators/api/api.go

```diff
@@ -45,13 +45,13 @@ type Operator interface {
 	// GenerateManifests generates manifests for the operator
 	GenerateManifests(*common.Cluster) (*Manifests, error)
 	// GetCPURequirementForWorker provides worker CPU requirements for the operator
-	GetCPURequirementForWorker(context.Context, *common.Cluster) (int64, error)
+	GetCPURequirementForWorker(*common.Cluster) (int64, error)
 	// GetCPURequirementForMaster provides master CPU requirements for the operator
-	GetCPURequirementForMaster(context.Context, *common.Cluster) (int64, error)
+	GetCPURequirementForMaster(*common.Cluster) (int64, error)
 	// GetMemoryRequirementForWorker provides worker memory requirements for the operator in MB
-	GetMemoryRequirementForWorker(ctx context.Context, cluster *common.Cluster) (int64, error)
+	GetMemoryRequirementForWorker(cluster *common.Cluster) (int64, error)
 	// GetMemoryRequirementForMaster provides master memory requirements for the operator in MB
-	GetMemoryRequirementForMaster(ctx context.Context, cluster *common.Cluster) (int64, error)
+	GetMemoryRequirementForMaster(cluster *common.Cluster) (int64, error)
```

Comment:
For consistency please either restore context parameter in the four methods above or remove it from `GetDisksRequirement*` methods

Reply:
Understood.